lib/util/option.js

```diff
@@ -38,28 +38,35 @@ class Option {
 }
 
 class Some extends Option {
-  #value;
+  // Should.js uses `for ... in` to compare objects, so while should.js is in
+  // use an Option's value property must be "enumerable".
+  //
+  // See: https://github.com/shouldjs/equal/blob/master/index.js
+  // See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Enumerability_and_ownership_of_properties
+  //
+  // TODO revert from __please_use_get to #value when should.js is removed.
+  //#value;
 
   /* istanbul ignore next */
   constructor(value) {
     super();
-    this.#value = value;
+    this.__please_use_get = value;
```

Comment:
I think the name `__please_use_get` makes the class as a whole a little hard to read. Could we try `_value` or `__value` and see if there's an issue? When the name was just `value`, I think it wasn't obvious that it wasn't meant to be exposed. However, I think that's much clearer with a name like `_value`.

Reply:
:+1: changed to `_value`